recipes/smudgeplot/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set version = "0.2.1" %}
+
+package:
+    name: smudgeplot
+    version: {{ version }}
+
+build:
+    number: 0
+
+source:
+    url: https://github.com/KamilSJaron/smudgeplot/archive/v{{ version }}.tar.gz
+    sha256: 2445564d12a852dcd5ade521b522cabde0df23512687b4ab5b46f73f9b315608
+
+requirements:
+    build:
+        - {{ compiler('c') }}
+        - make
+        - r-devtools
+        - r-base
+        - r-viridis
+        - r-argparse
+    host:
+        - python >=3.6
+        - r-base
+        - r-viridis
+        - r-argparse
+        - numpy
+        - scipy
+    run:
+        - python >=3.6
+        - r-base
+        - r-viridis
+        - r-argparse
+        - numpy
+        - scipy
+
+test:
+    commands:
+        - smudgeplot.py --version 2>&1 | grep 'Running'
+
+about:
+    home: https://github.com/KamilSJaron/smudgeplot
+    license: Apache-2.0
```

Comment:
I think the license is part of the tarball, so remove the license you manually added and add a `license_file: LICENSE` line.